docs/advanced-topics/best-practices/best-practices-dockstore.rst

```diff
@@ -0,0 +1,111 @@
+
+Another tip was already mentioned above; host your repositories on a non-personal organization. Similarly, try to use a non-personal namespace to register your Docker images.
+Doing this will group your tools/workflows together under our ``Tool: Namespace`` and ``Workflow: Organization`` facets.
+
+You should also consider adding labels to your entry since ``Labels`` is another facet on the search page. You can do this by going to `My Tools <https://www.dockstore.org/my-tools>`_ or `My Workflows <https://www.dockstore.org/my-workflows>`_ page.
+On the right hand side, above the tabs, you will see the text ``Manage labels``.
+
+.. image:: /assets/images/docs/manage-labels.png
+
+Once you click the text, you'll be able to add or remove labels for your entry. To get ideas for what to use as labels, look at what others are using on the search page.
+Also note that each label cannot have spaces and must be all lowercase. If you want a label be multiple words, separate them by hypen instead of spaces.
+
+You can also add a checker workflow to your tools and workflows to make use of our ``Has Checker Workflows`` facet. Checker workflows also guarantee that your entry,
+given some input, produces the expected output on a platform different from the one where you are developing. Read our :doc:`Checker Workflows <../checker-workflows>` tutorial to learn more.
+
+Another option for making your work more visible is using our :doc:`Organization and Collection <../organizations-and-collections>` pages.
+An Organization is a landing page for your collaborations, institutions, consortiums, companies, etc. Here you can explain the work and goals your group has,
+and highlight your tools and workflows by adding them to a Collection.
+
+Making It Easier to Understand
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Once a user has found your entry, they need to understand what it does and how to use it! The most important thing you can do so others understand your work is to provide a thorough description.
+You can do this by filling out the metadata field as explained in the best practices tutorials linked above, or you can write one in the README.md file.
+Your description, using either method, can be formatted using markdown. Once registered, it will be parsed by Dockstore and made available on the Info tab of an entry.
+Because it will be one the of first things a user will see when looking at your entry, you should make it as detailed as possible. Here is a list of items to write about:
+
+* About Section.
+
+   * What does your tool or workflow do?
+   * Are you part of a bigger organization? What are some of their goals?
+
+* How to Use Section.
+
+   * What are the system requirements? Minimum and recommended
+   * Describe the input and output files
+
+      * What are their names?
+      * What data do they contain?
+      * What is the format?
```

Comment:
Can point out that in CWL you can include this info in the descriptor which is even better because search understands it
https://www.commonwl.org/user_guide/16-file-formats/index.html